css-two-column-layout/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row wrap space-around">
+      <h1>Scale, Speed, Agility</h1>
+      <p id="headingtext">We've dedicated thousands of man hours to making healthcare institutions
+        in the Helium <br /> network run 10x more efficiently and deliver an unparalleled
+        quality of care.
+      </p>
```

Comment:
Since this content is a child of a `row` class, it should be in an element with a `column` class. It is full width, so an appropriate one would be `column-full`